raiden/tests/conftest.py

```diff
@@ -49,9 +49,9 @@ def pytest_addoption(parser):
 
     parser.addoption(
         "--transport",
-        choices=("none", "udp", "matrix", "all"),
+        choices=("none", "matrix"),
```

Comment:
The question is if we still need this anymore. Since there is only one version of the integration tests now the normal pytest selection methods should be sufficient.